src/docs/user/references/ScriptLanguageSupport.adoc

```diff
@@ -361,7 +361,7 @@ When an internal PowerShell type needs to be used by another language than Power
 
 === Perl
 
-The Perl script engines features the following behaviors:
+The Perl script engine features the following behaviors:
```

Comment:
features => has 
features goes more for name, adv more than verb

Reply:
@mboussaa features is the verb, no need to add `has`.